src/bo4e_generator/schema.py

```diff
@@ -70,10 +70,15 @@ def get_namespace(input_directory: Path) -> dict[str, SchemaMetadata]:
     return namespace
 
 
-def get_version(namespace: dict[str, SchemaMetadata]) -> str:
+def get_version(target_version: Optional[str], namespace: dict[str, SchemaMetadata]) -> str:
     """
     Get the version of the bo4e schemas.
     """
+    if target_version is not None:
+        gh_version_regex = re.compile(r"^v((?:\d+\.)*\d+)(?:-(rc\d+))?$")
+        if gh_version_regex.match(target_version) is not None:
+            target_version = gh_version_regex.sub(r"\1\2", target_version)
```

Comment:
named groups helfen imo bei regex mit der readability.

```suggestion
        gh_version_regex = re.compile(r"^v(?P<version>(?:\d+\.)*\d+)(?:-(?P<release_candidate)rc\d+))?$")
        if gh_version_regex.match(target_version) is not None:
            target_version = gh_version_regex.sub(r"\g<version>\g<release_candidate>", target_version)
```